src/konnektor/network_planners/generators/_abstract_network_generator.py

```diff
@@ -24,32 +24,34 @@ def __init__(
         self,
         mappers: Union[AtomMapper, list[AtomMapper]],
         scorer,
```

Comment:
Probably should type hint these with `Callable`?

Reply:
done, and opened an issue [here](https://github.com/OpenFreeEnergy/konnektor/issues/98) for type hinting to avoid scope-creep